modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -6,12 +6,6 @@ import {
 } from 'elementor-frontend-utils/flex-horizontal-scroll';
 
 export default class NestedTabs extends Base {
-	constructor( ...args ) {
-		super( ...args );
```

Comment:
We dont need anything from the base?

Reply:
We inherit the default constructor by default. We only need to add it, if we want to add properties to the constructor.